u'{"time":"2022-03-30 11:51:12,524","name":"root","level": "WARNING","module": "schema","line": "91","message": "employee does not exist"}'
u'{"time":"2022-03-30 11:51:12,564","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 92, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-03-30 11:51:51,156","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-03-30 11:51:51,168","name":"root","level": "INFO","module": "schema","line": "60","message": "created employee successfully"}'
u'{"time":"2022-03-30 11:52:06,874","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-03-30 11:52:06,878","name":"root","level": "ERROR","module": "schema","line": "64","message": "problem in creating employee"}'
u'{"time":"2022-03-30 11:52:06,880","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 65, in mutate
    raise Exception("problem in creating employee")
graphql.error.located_error.GraphQLLocatedError: problem in creating employee
"}'
u'{"time":"2022-03-30 11:53:02,042","name":"root","level": "WARNING","module": "schema","line": "133","message": "employee does not exist"}'
u'{"time":"2022-03-30 11:53:02,043","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 134, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-03-30 11:53:42,430","name":"root","level": "INFO","module": "schema","line": "137","message": "started deleting employee"}'
u'{"time":"2022-03-30 11:53:42,440","name":"root","level": "INFO","module": "schema","line": "139","message": "deleted employee successfully"}'
u'{"time":"2022-03-30 11:53:51,046","name":"root","level": "WARNING","module": "schema","line": "133","message": "employee does not exist"}'
u'{"time":"2022-03-30 11:53:51,048","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 134, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-03-30 11:59:12,395","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-03-30 11:59:12,413","name":"root","level": "INFO","module": "schema","line": "60","message": "created employee successfully"}'
u'{"time":"2022-03-30 11:59:16,316","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-03-30 11:59:16,332","name":"root","level": "ERROR","module": "schema","line": "64","message": "problem in creating employee"}'
u'{"time":"2022-03-30 11:59:16,333","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 65, in mutate
    raise Exception("problem in creating employee")
graphql.error.located_error.GraphQLLocatedError: problem in creating employee
"}'
u'{"time":"2022-03-30 12:00:33,307","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-03-30 12:00:33,314","name":"root","level": "ERROR","module": "schema","line": "64","message": "problem in creating employee"}'
u'{"time":"2022-03-30 12:00:33,317","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 65, in mutate
    raise Exception("problem in creating employee")
graphql.error.located_error.GraphQLLocatedError: problem in creating employee
"}'
u'{"time":"2022-03-30 12:00:50,975","name":"root","level": "WARNING","module": "schema","line": "133","message": "employee does not exist"}'
u'{"time":"2022-03-30 12:00:50,977","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 134, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-03-30 12:00:55,263","name":"root","level": "INFO","module": "schema","line": "137","message": "started deleting employee"}'
u'{"time":"2022-03-30 12:00:55,274","name":"root","level": "INFO","module": "schema","line": "139","message": "deleted employee successfully"}'
u'{"time":"2022-04-01 16:21:00,390","name":"root","level": "INFO","module": "schema","line": "45","message": "started creating employee"}'
u'{"time":"2022-04-01 16:21:00,420","name":"root","level": "INFO","module": "schema","line": "60","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:21:00,426","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "graphql.error.base.GraphQLError: Cannot return null for non-nullable field EmployeeType.empID.
"}'
u'{"time":"2022-04-01 16:23:54,864","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
graphql.error.located_error.GraphQLLocatedError: mutate() got an unexpected keyword argument 'empID'
"}'
u'{"time":"2022-04-01 16:25:04,713","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:25:04,738","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:25:13,882","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:25:13,892","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:26:17,228","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:26:17,251","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:33:36,692","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:33:36,697","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:35:00,901","name":"root","level": "INFO","module": "schema","line": "99","message": "started updating employee"}'
u'{"time":"2022-04-01 16:35:00,919","name":"root","level": "INFO","module": "schema","line": "114","message": "updated employee successfully"}'
u'{"time":"2022-04-01 16:35:18,117","name":"root","level": "INFO","module": "schema","line": "99","message": "started updating employee"}'
u'{"time":"2022-04-01 16:35:18,120","name":"root","level": "INFO","module": "schema","line": "114","message": "updated employee successfully"}'
u'{"time":"2022-04-01 16:35:49,902","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:35:49,925","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 16:37:59,801","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-01 16:37:59,827","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-01 17:00:18,404","name":"root","level": "WARNING","module": "schema","line": "136","message": "employee does not exist"}'
u'{"time":"2022-04-01 17:00:18,472","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 137, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-04-01 17:00:26,863","name":"root","level": "INFO","module": "schema","line": "140","message": "started deleting employee"}'
u'{"time":"2022-04-01 17:00:26,877","name":"root","level": "INFO","module": "schema","line": "142","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:39:01,399","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:39:01,429","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:42:46,954","name":"root","level": "INFO","module": "schema","line": "99","message": "started updating employee"}'
u'{"time":"2022-04-04 08:42:46,957","name":"root","level": "INFO","module": "schema","line": "105","message": "updated empDOJ successfully"}'
u'{"time":"2022-04-04 08:42:47,021","name":"root","level": "INFO","module": "schema","line": "120","message": "updated employee successfully"}'
u'{"time":"2022-04-04 08:45:54,179","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:45:54,190","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:46:05,412","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:46:05,424","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:46:43,223","name":"root","level": "INFO","module": "schema","line": "146","message": "started deleting employee"}'
u'{"time":"2022-04-04 08:46:43,231","name":"root","level": "INFO","module": "schema","line": "148","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:47:30,010","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:47:30,026","name":"root","level": "INFO","module": "schema","line": "63","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:49:46,455","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:49:46,456","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:49:46,465","name":"root","level": "INFO","module": "schema","line": "64","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:50:49,730","name":"root","level": "INFO","module": "schema","line": "149","message": "started deleting employee"}'
u'{"time":"2022-04-04 08:50:49,740","name":"root","level": "INFO","module": "schema","line": "151","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:51:29,158","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:51:29,159","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:51:29,159","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:51:29,160","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:51:29,169","name":"root","level": "INFO","module": "schema","line": "66","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:53:46,040","name":"root","level": "INFO","module": "schema","line": "150","message": "started deleting employee"}'
u'{"time":"2022-04-04 08:53:46,049","name":"root","level": "INFO","module": "schema","line": "152","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:54:01,023","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:54:01,024","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:54:01,025","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:54:01,027","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:54:01,039","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-04"}'
u'{"time":"2022-04-04 08:54:01,040","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:56:02,609","name":"root","level": "INFO","module": "schema","line": "150","message": "started deleting employee"}'
u'{"time":"2022-04-04 08:56:02,618","name":"root","level": "INFO","module": "schema","line": "152","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:56:14,976","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:56:14,977","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:56:14,979","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:56:14,979","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:56:14,990","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-04"}'
u'{"time":"2022-04-04 08:56:14,991","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:59:17,270","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:59:17,272","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:17,273","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:17,273","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:17,285","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:17,287","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 08:59:38,747","name":"root","level": "INFO","module": "schema","line": "150","message": "started deleting employee"}'
u'{"time":"2022-04-04 08:59:38,755","name":"root","level": "INFO","module": "schema","line": "152","message": "deleted employee successfully"}'
u'{"time":"2022-04-04 08:59:46,673","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 08:59:46,674","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:46,676","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:46,676","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 08:59:46,697","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-04"}'
u'{"time":"2022-04-04 08:59:46,697","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 09:00:43,189","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 09:00:43,190","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-02"}'
u'{"time":"2022-04-04 09:00:43,191","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-02"}'
u'{"time":"2022-04-04 09:00:43,192","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-02"}'
u'{"time":"2022-04-04 09:00:43,207","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-02"}'
u'{"time":"2022-04-04 09:00:43,208","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 09:01:01,450","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 09:01:01,455","name":"root","level": "INFO","module": "schema","line": "47","message": "None"}'
u'{"time":"2022-04-04 09:01:01,467","name":"root","level": "ERROR","module": "schema","line": "71","message": "problem in creating employee"}'
u'{"time":"2022-04-04 09:01:01,497","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 72, in mutate
    raise Exception("problem in creating employee")
graphql.error.located_error.GraphQLLocatedError: problem in creating employee
"}'
u'{"time":"2022-04-04 09:01:44,154","name":"root","level": "INFO","module": "schema","line": "46","message": "started creating employee"}'
u'{"time":"2022-04-04 09:01:44,156","name":"root","level": "INFO","module": "schema","line": "47","message": "2022-04-04"}'
u'{"time":"2022-04-04 09:01:44,157","name":"root","level": "INFO","module": "schema","line": "54","message": "2022-04-04"}'
u'{"time":"2022-04-04 09:01:44,157","name":"root","level": "INFO","module": "schema","line": "55","message": "2022-04-04"}'
u'{"time":"2022-04-04 09:01:44,165","name":"root","level": "INFO","module": "schema","line": "66","message": "2022-04-04"}'
u'{"time":"2022-04-04 09:01:44,166","name":"root","level": "INFO","module": "schema","line": "67","message": "created employee successfully"}'
u'{"time":"2022-04-04 11:53:13,553","name":"root","level": "WARNING","module": "schema","line": "142","message": "employee does not exist"}'
u'{"time":"2022-04-04 11:53:13,620","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 143, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-04-07 11:17:09,829","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 42, in wrapper
    raise exc
graphql.error.located_error.GraphQLLocatedError: You do not have permission to perform this action
"}'
u'{"time":"2022-04-07 18:04:03,859","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 41, in wrapper
    return f(*args, **kwargs)
graphql.error.located_error.GraphQLLocatedError: mutate() missing 1 required positional argument: 'empDOJ'
"}'
u'{"time":"2022-04-07 18:05:34,005","name":"root","level": "INFO","module": "schema","line": "63","message": "started creating employee"}'
u'{"time":"2022-04-07 18:05:34,023","name":"root","level": "INFO","module": "schema","line": "78","message": "created employee successfully"}'
u'{"time":"2022-04-07 18:19:13,021","name":"root","level": "INFO","module": "schema","line": "163","message": "started deleting employee"}'
u'{"time":"2022-04-07 18:19:13,027","name":"root","level": "INFO","module": "schema","line": "165","message": "deleted employee successfully"}'
u'{"time":"2022-04-07 18:19:25,704","name":"root","level": "INFO","module": "schema","line": "163","message": "started deleting employee"}'
u'{"time":"2022-04-07 18:19:25,714","name":"root","level": "INFO","module": "schema","line": "165","message": "deleted employee successfully"}'
u'{"time":"2022-04-07 18:19:27,900","name":"root","level": "WARNING","module": "schema","line": "159","message": "employee does not exist"}'
u'{"time":"2022-04-07 18:19:27,957","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 41, in wrapper
    return f(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 160, in mutate
    raise Exception("employee does not exist")
graphql.error.located_error.GraphQLLocatedError: employee does not exist
"}'
u'{"time":"2022-04-07 18:21:43,376","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 41, in wrapper
    return f(*args, **kwargs)
graphql.error.located_error.GraphQLLocatedError: mutate() missing 2 required positional arguments: 'empDOJ' and 'empDescription'
"}'
u'{"time":"2022-04-07 18:23:26,249","name":"root","level": "INFO","module": "schema","line": "63","message": "started creating employee"}'
u'{"time":"2022-04-07 18:23:26,266","name":"root","level": "INFO","module": "schema","line": "78","message": "created employee successfully"}'
u'{"time":"2022-04-07 18:23:52,634","name":"root","level": "INFO","module": "schema","line": "63","message": "started creating employee"}'
u'{"time":"2022-04-07 18:23:52,641","name":"root","level": "INFO","module": "schema","line": "78","message": "created employee successfully"}'
u'{"time":"2022-04-07 18:24:24,867","name":"root","level": "INFO","module": "schema","line": "163","message": "started deleting employee"}'
u'{"time":"2022-04-07 18:24:24,875","name":"root","level": "INFO","module": "schema","line": "165","message": "deleted employee successfully"}'
u'{"time":"2022-04-07 18:38:39,120","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 42, in wrapper
    raise exc
graphql.error.located_error.GraphQLLocatedError: You do not have permission to perform this action
"}'
u'{"time":"2022-04-08 11:21:08,322","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 30, in wrapper
    return func(info.context, *args, **kwargs)
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql_jwt\decorators.py", line 42, in wrapper
    raise exc
graphql.error.located_error.GraphQLLocatedError: You do not have permission to perform this action
"}'
u'{"time":"2022-04-08 12:05:54,991","name":"root","level": "INFO","module": "schema","line": "61","message": "started creating employee"}'
u'{"time":"2022-04-08 12:05:54,999","name":"root","level": "INFO","module": "schema","line": "76","message": "created employee successfully"}'
u'{"time":"2022-04-08 16:44:09,285","name":"root","level": "INFO","module": "schema","line": "61","message": "started creating employee"}'
u'{"time":"2022-04-08 16:44:09,309","name":"root","level": "INFO","module": "schema","line": "76","message": "created employee successfully"}'
u'{"time":"2022-04-08 17:09:23,945","name":"root","level": "INFO","module": "schema","line": "158","message": "started deleting employee"}'
u'{"time":"2022-04-08 17:09:23,968","name":"root","level": "INFO","module": "schema","line": "160","message": "deleted employee successfully"}'
u'{"time":"2022-04-08 17:11:13,952","name":"root","level": "INFO","module": "schema","line": "61","message": "started creating employee"}'
u'{"time":"2022-04-08 17:11:13,976","name":"root","level": "INFO","module": "schema","line": "76","message": "created employee successfully"}'
u'{"time":"2022-04-08 17:11:39,151","name":"root","level": "INFO","module": "schema","line": "158","message": "started deleting employee"}'
u'{"time":"2022-04-08 17:11:39,172","name":"root","level": "INFO","module": "schema","line": "160","message": "deleted employee successfully"}'
u'{"time":"2022-04-08 17:13:05,885","name":"root","level": "INFO","module": "schema","line": "61","message": "started creating employee"}'
u'{"time":"2022-04-08 17:13:05,899","name":"root","level": "INFO","module": "schema","line": "76","message": "created employee successfully"}'
u'{"time":"2022-04-08 17:13:23,754","name":"root","level": "INFO","module": "schema","line": "158","message": "started deleting employee"}'
u'{"time":"2022-04-08 17:13:23,776","name":"root","level": "INFO","module": "schema","line": "160","message": "deleted employee successfully"}'
u'{"time":"2022-04-08 17:16:23,822","name":"root","level": "INFO","module": "schema","line": "61","message": "started creating employee"}'
u'{"time":"2022-04-08 17:16:23,845","name":"root","level": "INFO","module": "schema","line": "76","message": "created employee successfully"}'
u'{"time":"2022-04-08 17:19:07,267","name":"root","level": "INFO","module": "schema","line": "158","message": "started deleting employee"}'
u'{"time":"2022-04-08 17:19:07,282","name":"root","level": "INFO","module": "schema","line": "160","message": "deleted employee successfully"}'
u'{"time":"2022-04-26 16:13:29,214","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
graphql.error.located_error.GraphQLLocatedError: resolve_employees() missing 1 required positional argument: 'empID'
"}'
u'{"time":"2022-04-26 16:15:51,435","name":"root","level": "INFO","module": "schema","line": "119","message": "started updating employee"}'
u'{"time":"2022-04-26 16:15:51,436","name":"root","level": "INFO","module": "schema","line": "122","message": "updated empName successfully"}'
u'{"time":"2022-04-26 16:15:51,436","name":"root","level": "INFO","module": "schema","line": "125","message": "updated empDOJ successfully"}'
u'{"time":"2022-04-26 16:15:51,437","name":"root","level": "INFO","module": "schema","line": "128","message": "updated empDescription successfully"}'
u'{"time":"2022-04-26 16:15:51,437","name":"root","level": "INFO","module": "schema","line": "131","message": "updated empCategory successfully"}'
u'{"time":"2022-04-26 16:15:51,438","name":"root","level": "INFO","module": "schema","line": "134","message": "updated empCity successfully"}'
u'{"time":"2022-04-26 16:15:51,438","name":"root","level": "INFO","module": "schema","line": "137","message": "updated empOfficeVenue successfully"}'
u'{"time":"2022-04-26 16:15:51,453","name":"root","level": "INFO","module": "schema","line": "140","message": "updated employee successfully"}'
u'{"time":"2022-04-26 16:16:21,930","name":"root","level": "INFO","module": "schema","line": "119","message": "started updating employee"}'
u'{"time":"2022-04-26 16:16:21,931","name":"root","level": "INFO","module": "schema","line": "122","message": "updated empName successfully"}'
u'{"time":"2022-04-26 16:16:21,931","name":"root","level": "INFO","module": "schema","line": "125","message": "updated empDOJ successfully"}'
u'{"time":"2022-04-26 16:16:21,932","name":"root","level": "INFO","module": "schema","line": "128","message": "updated empDescription successfully"}'
u'{"time":"2022-04-26 16:16:21,933","name":"root","level": "INFO","module": "schema","line": "131","message": "updated empCategory successfully"}'
u'{"time":"2022-04-26 16:16:21,934","name":"root","level": "INFO","module": "schema","line": "134","message": "updated empCity successfully"}'
u'{"time":"2022-04-26 16:16:21,941","name":"root","level": "INFO","module": "schema","line": "137","message": "updated empOfficeVenue successfully"}'
u'{"time":"2022-04-26 16:16:21,946","name":"root","level": "INFO","module": "schema","line": "140","message": "updated employee successfully"}'
u'{"time":"2022-04-26 16:18:46,503","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 208, in mutate
    raise Exception('User Already Exists!')
graphql.error.located_error.GraphQLLocatedError: User Already Exists!
"}'
u'{"time":"2022-04-26 16:18:56,308","name":"graphql.execution.utils","level": "ERROR","module": "utils","line": "155","message": "Traceback (most recent call last):
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 489, in _resolve_from_executor
    executor(resolve, reject)
  File "D:\1_bcg\django_app\venv\lib\site-packages\promise\promise.py", line 756, in executor
    return resolve(f(*args, **kwargs))
  File "D:\1_bcg\django_app\venv\lib\site-packages\graphql\execution\middleware.py", line 75, in make_it_promise
    return next(*args, **kwargs)
  File "D:\1_bcg\django_app\Employee_Project\Employee_App\schema.py", line 208, in mutate
    raise Exception('User Already Exists!')
graphql.error.located_error.GraphQLLocatedError: User Already Exists!
"}'
